lib/index.js

```diff
@@ -3,5 +3,6 @@ import FyllUtRouter from "../src/Forms/FyllUtRouter";
 import Template from "../src/template";
 import { AppConfigProvider } from "../src/configContext";
 import AmplitudeProvider from "../src/context/amplitude";
+import { globalStyles, appStyles } from "../src/components/navGlobalStyles";
 
-export { CustomComponents, FyllUtRouter, Template, AppConfigProvider, AmplitudeProvider };
+export { CustomComponents, FyllUtRouter, Template, AppConfigProvider, AmplitudeProvider, globalStyles, appStyles };
```

Comment:
No need to fix now, but I think one import/export called "styles" will be scale better. Should change if we add another grouping later on.